app/controllers/app.controllers.js

```diff
@@ -0,0 +1,8 @@
+const { fetchTopics } = require('../models/app.models')
+
+exports.getTopics = (request, response)=>{
+    fetchTopics()
+    .then((result)=>{
+        response.status(200).send(result)
```

Comment:
you should be sending data back as a property value within an object - {topics : result} for example